style.css

```diff
@@ -11,6 +11,23 @@ html {
     font-family: Helvetica, sans-serif;   
     text-align: center; 
 }
+.banner-container h2 {
+    font-size: xx-large;
+    text-align: center;
+    background-color: #5471ab;
+    padding: 5%
+}
+
+.banner-container img {
+    width: 100%;
+    padding: 0%;
+    margin: 0;
+    border-width: 1px;
+    max-width: 100%;
+    background-color: #5471ab;
+    height: auto;
+    background-image: url("images/image_banner.png");
```

Comment:
please move images into folder resources/images/

Reply:
I can't do this until I have the version with the folder in it

Reply:
well, you can create the folder, sorry for misleading